db/schema.rb

```diff
@@ -82,6 +82,7 @@
     t.datetime "updated_at", null: false
     t.string "title"
     t.string "description", default: "", null: false
+    t.integer "visibility_level", limit: 2, default: 0, null: false, comment: "Used as enum in Rails"
```

Comment:
In the migration, `limit` is 1, but here it is 2. 🤔 

And why is the schema timestamp not updated?

Reply:
The changed limit is interesting, setting limit for integers specifies the number of bytes that the number uses. Since Postgres does not have a `tinyint` type it reverts to 2, so it can fit a `smallint`.

the timestamp probably went MIA during the merge